test/it/code-samples/positive/public/importJson.js

```diff
@@ -1 +1,3 @@
-import data from 'someFiles.json'
```

Comment:
consider splitting/renaming this code to a dedicated test (public-import-json) as you did in pages and backend.

Reply:
Done